conf/application.conf

```diff
@@ -348,13 +348,25 @@ db {
   # https://www.playframework.com/documentation/latest/Highlights25#Logging-SQL-statements
   #default.logSql=true
 }
-play.http.secret.key = "changeme"
 
+my.executor {
+  fork-join-executor {
+    parallelism-factor = 20.0
+    parallelism-max = 200
+  }
+}
+
+#play.http.secret.key = "changeme"
+play.filters.enabled += "play.filters.headers.SecurityHeadersFilter"
+#play.filters.headers.contentSecurityPolicy = null
 play.i18n.langs = [ "en" ]
 
 play.modules.enabled += "module.Module"
-play.http.errorHandler = "utils.ErrorHandler"
+# play.http.errorHandler = "utils.ErrorHandler"
```

Comment:
Why was the error handler commented out?

Reply:
the error handler was not yet modified for the newly created concept, namely not using scala play to define the views but to use an angular application as well as the corresponding changes on the server. 
I kept the error handler als well as this definition of it in the application.conf uncommented here as a reference of the old implementation, which we need to alter to fit the new code.
This is planned to be part of the next milestone.

Reply:
Okay, cool.